sim/ofat.py

```diff
@@ -44,22 +46,25 @@ def ofat(btb_seq_path, results_path, reference_path, branches=DEFAULT_BRANCHES):
     # Prepare output directory
     os.makedirs(results_path, exist_ok=False)
 
+    samples = validator.standard_samples()
```

Comment:
I have this feeling like it makes more sense to setup the standard samples in this file rather than in validator.py/ It just feels a little bit more general to me, like this is where we setup all the test parammaters e.g. the branches and also what samples to use.

Validator is then a more general module encompassing the validator pipeline. 

Maybe just personal preference so happy to concede.